### PR TITLE
adding kuberhealthy

### PR DESCRIPTION
**Pull request by integrii**
Signed-off-by: Eric Greer <eric.greer@comcast.com>

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by viglesiasce — CHANGES_REQUESTED** [review]
Need to disable the Prometheus part by default since you can't guarantee the CRDs (ServiceMonitor) are installed in the cluster

**Review by viglesiasce — APPROVED** [review]

File: stable/kuberhealthy/templates/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
```
Comment (viglesiasce) [inline stable/kuberhealthy/templates/namespace.yaml]: Remove this namespace

File: stable/kuberhealthy/NOTES.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+Kuberhealthy has been installed to your cluster!
+
+	Docker Image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+	Namespace: {{ .Values.namespace }}
```
Comment (viglesiasce) [inline stable/kuberhealthy/NOTES.txt]:
```suggestion
	Namespace: {{ .Release.Namespace }}
```

File: stable/kuberhealthy/templates/clusterrolebinding.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: {{ template "rbac.apiVersion" . }}
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "kuberhealthy.name" . }}
+  namespace: {{ .Values.deployment.namespace }}
```
Comment (viglesiasce) [inline stable/kuberhealthy/templates/clusterrolebinding.yaml]:
```suggestion
```

File: stable/kuberhealthy/templates/configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+{{- if .Values.prometheus.enableAlerting -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: "prometheus"
+    prometheus: {{ .Values.prometheus.name }}
+    role: alert-rules
+  name: {{ template "kuberhealthy.name" . }}
+  namespace: {{ .Values.deployment.namespace }}
```
Comment (viglesiasce) [inline stable/kuberhealthy/templates/configmap.yaml]:
```suggestion
```

File: stable/kuberhealthy/templates/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+  labels:
+    app: {{ template "kuberhealthy.name" . }}
+    chart: {{ .Chart.Name }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  namespace: {{ .Values.deployment.namespace }}
```
Comment (viglesiasce) [inline stable/kuberhealthy/templates/deployment.yaml]:
```suggestion
```

File: stable/kuberhealthy/templates/poddisruptionbudget.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name:  {{ template "kuberhealthy.name" . }}-pdb
+  namespace: {{ .Values.deployment.namespace }}
```
Comment (viglesiasce) [inline stable/kuberhealthy/templates/poddisruptionbudget.yaml]:
```suggestion
```

File: stable/kuberhealthy/templates/role.yaml [hunk 7]
```diff
@@ -0,0 +1,72 @@
+apiVersion: {{ template "rbac.apiVersion" . }}
+kind: Role
+metadata:
+  name: {{ template "kuberhealthy.name" . }}
+  namespace: {{ .Values.deployment.namespace }}
```
Comment (viglesiasce) [inline stable/kuberhealthy/templates/role.yaml]:
```suggestion
```

File: stable/kuberhealthy/templates/rolebinding.yaml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+apiVersion: {{ template "rbac.apiVersion" . }}
+kind: RoleBinding
+metadata:
+  name: {{ template "kuberhealthy.name" . }}
+  namespace: {{ .Values.deployment.namespace }}
```
Comment (viglesiasce) [inline stable/kuberhealthy/templates/rolebinding.yaml]:
```suggestion
```

File: stable/kuberhealthy/templates/service.yaml [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+  {{ if .Values.prometheus.enabled -}}
+  {{ if .Values.prometheus.enableScraping -}}
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: {{ .Values.service.externalPort | quote }}
+    prometheus.io/path: "/metrics"
+  {{ end -}}
+  {{ end -}}
+  name: {{ template "kuberhealthy.name" . }}
+  namespace: {{ .Values.deployment.namespace }}
```
Comment (viglesiasce) [inline stable/kuberhealthy/templates/service.yaml]:
```suggestion
```

File: stable/kuberhealthy/values.yaml [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+tolerations:
+  # change to true to tolerate and deploy to masters
+  master: false
+
+deployment:
+  replicas: 2
+  maxSurge: 0
+  maxUnavailable: 1
+  imagePullPolicy: IfNotPresent
+  namespace: kuberhealthy
```
Comment (viglesiasce) [inline stable/kuberhealthy/values.yaml]:
```suggestion
```

**Comment by integrii** [comment]
It was good talking with you today @viglesiasce.  I actually want to try a pixelbook now.

Looks like the most recent build error involves the CRD already existing in the test cluster?

```
I1213 03:56:24.977] Error: release kuberhealthy-1q9tq4e294 failed: customresourcedefinitions.apiextensions.k8s.io "khstates.comcast.github.io" already exists
```

Perhaps I will wait a few and give it a retest.

**Comment by integrii** [comment]
I think it's ready for a LGTM?!  @viglesiasce

File: stable/kuberhealthy/README.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+prometheus:
+  enabled: true # do we deploy a ServiceMonitor spec?
+  name: "prometheus" # the name of the Prometheus deployment in your environment.
+  enableScraping: true # add the Prometheus scrape annotation to Kuberhealthy pods
+  serviceMonitor: true # use a ServiceMonitor configuration
+  enableAlerting: true # enable default Kuberhealthy alerts configuration
+app:
+  name: "kuberhealthy" # what to name the kuberhealthy deployment
+  namespace: "kuberhealthy" # what namespace to deploy the kuberhealthy app into
+  namespace: "kuberhealthy" # what namespace to deploy the kuberhealthy app into
```
Comment (philoserf) [inline stable/kuberhealthy/README.md]: `namespace` appears twice.

File: stable/kuberhealthy/Chart.yaml [hunk 12]
```diff
@@ -0,0 +1,22 @@
...
+keywords:
+- kuberhealthy
+- kubernetes
+- monitoring
+- synthetic
+- comcast
+- health
+- cluster
+sources:
+- https://github.com/Comcast/kuberhealthy
```
Comment (philoserf) [inline stable/kuberhealthy/Chart.yaml:21]: There can be multiple `sources`. Including the chart is useful, particularly in <https://hub.helm.sh>

File: stable/kuberhealthy/README.md (hunk 11 above)
Comment (philoserf) [reply stable/kuberhealthy/README.md]: `namespace` is not in the values file.

**Review by philoserf — CHANGES_REQUESTED** [review]

File: stable/kuberhealthy/Chart.yaml (hunk 12 above)
Comment (viglesiasce) [reply stable/kuberhealthy/Chart.yaml:21]: Added.

**Review by viglesiasce — APPROVED** [review]

**Pull request merged**